clang/docs/LanguageExtensions.rst

```diff
@@ -434,6 +434,37 @@ __datasizeof
 ``__datasizeof`` behaves like ``sizeof``, except that it returns the size of the
 type ignoring tail padding.
 
+.. _builtin_structured_binding_size-doc:
+
+__builtin_structured_binding_size (C++)
+---------------------------------------
+``__builtin_structured_binding_size`` returns the *structured binding size*
```

Comment:
I don't have a better idea, but this name does NOT imply that.  I read it as "it returns the size of a thing that is already a structured binding".  Maybe something like destructured_size or something goofy?  IDK, too early in the morning to be creative.

Reply:
I wasn't inspired so I went with the standard https://eel.is/c++draft/dcl.struct.bind#2